src/app/builders/components/Table/BuilderDataRow.tsx

```diff
@@ -130,19 +123,19 @@ export const BuilderCell = (props: BuilderNameCellProps): ReactElement => {
 }
 
 // TODO: @refactor move to app/components/Table/Cell/TableCell.tsx
-export const TableCellBase = <CID extends BaseColumnId = BaseColumnId>({
```

Comment:
I guess that this does not need to be exported since is no longer use in other components like the `BackerRewardsTable`